lib/widgets/action_sheet.dart

```diff
@@ -39,12 +39,12 @@ void showMessageActionSheet({required BuildContext context, required Message mes
       return Column(children: [
         if (!hasThumbsUpReactionVote) AddThumbsUpButton(message: message, messageListContext: context),
         StarButton(message: message, messageListContext: context),
-        ShareButton(message: message, messageListContext: context),
         if (isComposeBoxOffered) QuoteAndReplyButton(
           message: message,
           messageListContext: context,
         ),
         CopyMessageTextButton(message: message, messageListContext: context),
+        ShareButton(message: message, messageListContext: context),
```

Comment:
```
action-sheet: Move the "Share" option to the bottom of options
```

Let's also move the 'ShareButton' group in the tests, so the tests still match the order of the code they're testing.